packages/echo-core/src/services/analytics/analyticsTelemetryFilter.ts

```diff
@@ -1,24 +1,25 @@
-import { ITelemetryItem } from '@microsoft/applicationinsights-web';
+import { ITelemetryItem } from '.';
 import { AnalyticsNameFilterFunction } from './analyticsTypes';
 
 let isLogToConsoleEnabled = false;
 let isLogExcludedToConsoleEnabled = false;
 const telemetryLogContext = '[Telemetry]';
 const nameFilterFunctions: AnalyticsNameFilterFunction[] = [] as AnalyticsNameFilterFunction[];
-export function telemetryFilterShouldInclude(item: ITelemetryItem): boolean {
-    const itemBaseDataName = item.baseData ? item.baseData['name'] : undefined;
+export function telemetryFilterShouldInclude(telemetryItem: ITelemetryItem): boolean {
+    const itemBaseDataName = telemetryItem.baseData ? telemetryItem.baseData['name'] : 'undefined';
```

Comment:
Why is `undefined` an actual string here?

Reply:
Yes, fixed now. Was testing something earlier, and forgot to change it back to undefined